custom-metrics-stackdriver-adapter/Dockerfile

```diff
@@ -12,10 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.13-alpine as builder
+FROM golang:1.16-alpine as builder
+ENV GO111MODULE=on
```

Comment:
This is redundant, that's the default behaviour in go 1.16

Reply:
Good catch, will remove `ENV GO111MODULE=on`